tools/accuracy_checker/configs/person-detection-0106.yml

```diff
@@ -0,0 +1,58 @@
+models:
+  - name: person-detection-0106
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/person-detection-0106/FP32/person-detection-0106.xml
+        weights: intel/person-detection-0106/FP32/person-detection-0106.bin
+        adapter:
+          type: mask_rcnn
+          boxes_out: boxes
```

Comment:
why you need mask_rcnn adapter if you have only boxes output?

Reply:
Two reasons here. First, is that some adapter is required. :smile: Second, for consistency. In OTE we currently export detection and instance segmentation models is a common way, so having a common adapter in AC looks reasonable to me.

Reply:
than `mask_rcnn` name looks confused :)